changes/ce/fix-13842.en.md

```diff
@@ -0,0 +1 @@
+Fixed and UTF-8 string validation exception.
```

Comment:
typo: and -> a

Reply:
pushed a fix directly without sending a PR to release-58